src/components/MultipleAvatars.js

```diff
@@ -124,11 +131,13 @@ const MultipleAvatars = (props) => {
                         {/* View is necessary for tooltip to show for multiple avatars in LHN */}
                         <View>
                             <Avatar
-                                source={props.icons[0] || props.fallbackIcon}
+                                source={props.icons[0].source || props.fallbackIcon}
```

Comment:
Should we use `lodashGet` here?

Reply:
I think it's unnecessary as none of the keys are deeply nested (i.e icons.source.url)

Reply:
Agreed with not being necessary, I guess if `props.icons[0]` is missing there is something else completely wrong